packages/example/pubspec.yaml

```diff
@@ -1,14 +1,13 @@
 name: google_ml_kit_example
-description: Demonstrates how to use the google_ml_kit plugin.
+description: "Demonstrates how to use the google_ml_kit plugin."
```

Comment:
why in some `pubspec.yaml` file you added double quotes? and in other single quotes? 
I would like to keep all of them unquoted.

https://stackoverflow.com/a/69850618/3610617

Reply:
The `flutter create` command added those only around description. Seems like some flutter standard or something.